src/Acts/CamdramBundle/Controller/ShowController.php

```diff
@@ -116,6 +116,13 @@ public function modifyEditForm($form, $identifier) {
             $soc = $soc instanceof Society ? $soc->getName() : $soc["name"];
         }
         $form->get('societies')->setData($socs);
+
+        // Venues
+        $form->get('multi_venue')->setData($show->getMultiVenue());
+        $performances = $show->getPerformances();
+        if (!$performances->isEmpty()) {
+            $form->get('venue')->setData($performances->first()->getVenueName());
```

Comment:
Assuming this is for backwards compatability with #536?

Reply:
No, this is needed to set up the content of the show edit form correctly. It's correctly setting the radio button and transferring the venue name into the "Venue" field in this example of a show with multiple performances at one venue:
![Screenshot from 2019-04-21 20-08-14](https://user-images.githubusercontent.com/5357642/56474407-3680bf00-6471-11e9-84b5-ecfe66d6f894.png)